plugins/modules/device_credential_workflow_manager.py

```diff
@@ -344,49 +416,49 @@
     config:
     - global_credential_details:
         cli_credential:
-        - description: string
-          username: string
-          password: string
-          enable_password: string
+        - description: CLI1
+          username: cli1
+          password: '12345'
+          enable_password: '12345'
         snmp_v2c_read:
-        - description: string
-          read_community: string
+        - description: SNMPv2c Read1
+          read_community: '123456'
         snmp_v2c_write:
-        - description: string
-          write_community: string
+        - description: SNMPv2c Write1
+          write_community: '123456'
         snmp_v3:
-        - auth_password: string
+        - auth_password: '12345678'
           auth_type: SHA
           snmp_mode: AUTHPRIV
-          privacy_password: string
+          privacy_password: '12345678'
           privacy_type: AES128
-          username: string
-          description: string
+          username: snmpV31
+          description: snmpV31
         https_read:
-        - description: string
-          username: string
-          password: string
+        - description: HTTP Read1
+          username: HTTP Read1
+          password: '12345'
           port: 443
         https_write:
-        - description: string
-          username: string
-          password: string
+        - description: HTTP Write1
+          username: HTTP Write1
+          password: '12345'
           port: 443
       assign_credentials_to_site:
         cli_credential:
-          id: string
+          id: '1b3777a0-09c2-488b-a64c-6f3c441e343'
```

Comment:
Why do we need to pass id here? IDs are not easy to remember. Can you give description?

Reply:
removed id and added description.